pkg/meta/redis.go

```diff
@@ -2235,7 +1984,7 @@ func (r *redisMeta) cleanupSlices() {
 							chunkid, _ := strconv.Atoi(ps[0][1:])
```

Comment:
Although "chunkid" has no change here, it is better to use "strconv.ParseUint(ps[0][1:], 10, 64)" instead of strconv.Atoi(ps[0][1:]) here.